ios/FluentUI.Demo/FluentUI.Demo/Demos/CardViewDemoController.swift

```diff
@@ -14,41 +14,43 @@ class CardViewDemoController: DemoController {
 
         let demoIcon = UIImage(named: "flag-24x24")
 
-        for style in CardStyle.allCases {
-            addTitle(text: style.description)
+        for size in CardSize.allCases {
+            addTitle(text: size.description)
 
-            switch style {
-            case .horizontal:
+            switch size {
+            case .small:
                 if let demoIcon = demoIcon {
-                    let card = CardView(style: style, title: "Title", icon: demoIcon, colorStyle: .neutral)
+                    let card = CardView(size: size, title: "Title", icon: demoIcon, colorStyle: .neutral)
```

Comment:
nit: can you break the parameters into multiple lines for readability? (here and in the other occurrences below)